src/main/webapp/testjsp/test.jsp

```diff
@@ -0,0 +1,36 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
```

Comment:
Yes. "t" for **t**eammates or **t**ag.
"t" appears to be pretty standard for custom tags, **i**nstructor is mainly for its ease of use (character length), but changing it is fine.
I don't think it'll be much of a cause for confusion either way.